_shows/17_18/and_another_thing.md

```diff
@@ -0,0 +1,39 @@
+---
+title: ...And Another Thing! An Improvised Play
+period: Spring
+company: The Red Herrings
+season: External
+season_sort: 170
+date_start: 2017-01-29
+date_end: 2017-02-02
+venue: New Theatre
+
+cast:
+- name: Ian Sheard
```

Comment:
Is everyone in cast and crew NT? (Not seeing any person: false)

Reply:
Was based off last year’s file. Can go through and do this for this year’s and last year’s but can’t be of use before then

Reply:
Good stuff 😄